tests/metrics/test-irloc.sh

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2317
+# TODO: REMOVE SHELLCHECK DISABLE RIGHT AFTER IMPLEMENTING irloc.sh
+# The MIT License (MIT)
+#
+# Copyright (c) 2021-2024 Yegor Bugayenko
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+set -e
+set -o pipefail
+
+# TODO: ENABLE THIS TESTS RIGHT AFTER IMPLEMENTING irloc.sh VIA REMOVING `exit 0`
+exit 0
+
+temp=$1
+stdout=$2
+
+{
+  tmp=$(mktemp -d /tmp/XXXX)
+  cd "${tmp}"
+  mkdir -p "${LOCAL}/${temp}"
+  touch "${LOCAL}/${temp}/file.java"
+  "${LOCAL}/metrics/irloc.sh" "${LOCAL}/${temp}/file.java" "${LOCAL}/${temp}/stdout"
+  grep "Provided non-git repo" "${LOCAL}/${temp}/stdout"
```

Comment:
@zaqbez39me I think the script should fail here, instead of printing a message to the console